packages/ui-scripts/lib/index.js

```diff
@@ -57,55 +53,31 @@ function listCommands() {
 // be added to the `commands` directory and should be executed via yargs by calling
 // this function instead of requiring the file directly.
 function executeYargs() {
-  /* eslint-disable no-unused-expressions */
-  require('yargs').commandDir('./commands').version(false).help().argv
-  /* eslint-enable no-unused-expressions */
+  // https://github.com/yargs/yargs/blob/main/docs/advanced.md#example-command-hierarchy-using-indexmjs
+  // eslint-disable-next-line no-unused-expressions
+  yargs(hideBin(process.argv)).command(yargCommands).argv
 }
 
 // Place yargs commands at the start so users can get yargs argument documentation
 // by setting the --help flag
-if (process.argv.includes('create-component')) {
+if (process.argv.includes('bump')) {
+  executeYargs()
+} else if (process.argv.includes('commit')) {
   executeYargs()
-} else if (process.argv.includes('create-from-template')) {
+} else if (process.argv.includes('lint-commit')) {
   executeYargs()
-} else if (process.argv.includes('create-package')) {
+} else if (process.argv.includes('server')) {
   executeYargs()
 } else if (process.argv.includes('--help')) {
   listCommands()
-} else if (process.argv.includes('--bump')) {
-  require('./bump')
 } else if (process.argv.includes('--publish')) {
-  require('./publish')
-} else if (process.argv.includes('--deploy-docs')) {
-  require('./deploy-docs')
-} else if (process.argv.includes('--deploy-docs-via-github')) {
-  require('./deploy-docs-via-github')
-} else if (process.argv.includes('--build')) {
-  require('./build')
-} else if (process.argv.includes('--test')) {
-  require('./test/karma')
-} else if (process.argv.includes('--commit')) {
-  require('./commit')
-} else if (process.argv.includes('--generate-package-list')) {
-  require('./generate-package-list')
-} else if (process.argv.includes('--link-packages')) {
-  require('./link-packages')
-} else if (process.argv.includes('--lint-commit')) {
-  require('./lint-commit')
-} else if (process.argv.includes('--lint')) {
-  require('./test/lint')
-} else if (process.argv.includes('--clean')) {
-  require('./build/clean')
-} else if (process.argv.includes('--install-react')) {
-  require('./install-react')
-} else if (process.argv.includes('--server')) {
-  require('./server')
```

Comment:
I have removed a lot from here. some because they are just alternatives to a different command (e.g. `ui-scripts --lint === ui-test --lint`), some never existed (deploy-docs and deploy-docs-via-github) and some were never used (install-react, link-packages)